apps/bcc3d.ca/assets/js/request.js

```diff
@@ -1,29 +1,60 @@
 ---
 ---
+const REGIONS_API_ENDPOINT = "https://www.qa.bcc3d.ca/api/v1/region";
+
+window.onload = () => {
+    fetch(REGIONS_API_ENDPOINT)
+        .then((response) => {
+            return response.json();
```

Comment:
lambda with a single return statement can have the braces and return omitted, e.g.
`.then(response => response.json())`